backends/portfolio/tasks.rake

```diff
@@ -152,7 +154,117 @@ def pf_adoc2html(adoc_file, target_pname)
   sh "chmod +x #{run_pname}"
 
   # Now run the actual command.
-  sh cmd
+  sh(cmd)
 
   $logger.info "Generated HTML in #{target_pname}"
 end
+
+# @param adoc_file [String] Full pathname of source adoc file
+# @param target_pname [String] Full name of tags file being generated
+# @param isa_manual_dirname [String] Full pathname of ISA manual root directory
+def pf_adoc2norm_tags(adoc_file, target_pname, isa_manual_dirname)
```

Comment:
It would be helpful to have at least a one-liner description of what this function does.